bin/build

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-find ./src -name '*.ls' --print0 | xargs -0 lsc --compile --bare --output dist/
+lsc --compile --bare --output dist src
```

Comment:
`--print0` is not posix compliant and `lsc` will already recursively compile if you give it a directory, so we don't need `xargs`